### PR TITLE
Ground work for Personality Assignments added.

### PR DESCRIPTION
**Pull request by thebestburrito**
It's real basic and kinda hard to understand, very sorry, yes.

### CODE REVIEW

File: PeopleSimulator/src/Personality.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        }
+    }
+
+    public void setPersonalityBoolean1(String pB1) {
+        boolean isLawful;
+        boolean isSocial;
+        boolean is1Neutral;
+        boolean isRebeboolean isChaotic;
+
+        if (Personality.getPeronalityNumber(Society.getPerson(i)) == 0 || Person.getPersonalityNumber == 5 || Person.getPersonalityNumber == 10 || Person.getPeronalityNumber == 15 || Person.getPeronalityNumber == 20) {
```
Comment (thebestburrito) [inline PeopleSimulator/src/Personality.java]: Honestly I'll just ask Mr. Wray about this stuff, how to access a specific person. I do not know.

**Comment by bradwray** [comment]
It had these errors when I went to run it...

Society.java:35: error: illegal start of expression
  public void cycleDay() {
  ^
Society.java:35: error: illegal start of expression
  public void cycleDay() {
         ^
Society.java:35: error: ';' expected
  public void cycleDay() {
                      ^
Society.java:41: error: illegal start of expression
  public void addPerson(Person p1, Person p2) {
  ^
Society.java:41: error: illegal start of expression
  public void addPerson(Person p1, Person p2) {
         ^
Society.java:41: error: ';' expected
  public void addPerson(Person p1, Person p2) {
                       ^
Society.java:41: error: ';' expected
  public void addPerson(Person p1, Person p2) {
                                         ^
Society.java:41: error: not a statement
  public void addPerson(Person p1, Person p2) {
                                          ^
Society.java:41: error: ';' expected
  public void addPerson(Person p1, Person p2) {
                                            ^
Society.java:69: error: illegal start of expression
  public static long bellCurve(int mean, int sd) {
  ^
Society.java:69: error: illegal start of expression
  public static long bellCurve(int mean, int sd) {
         ^
Society.java:69: error: ';' expected
  public static long bellCurve(int mean, int sd) {
               ^
Society.java:69: error: '.class' expected
  public static long bellCurve(int mean, int sd) {
                                   ^
Society.java:69: error: ';' expected
  public static long bellCurve(int mean, int sd) {
                                       ^
Society.java:69: error: ';' expected
  public static long bellCurve(int mean, int sd) {
                                               ^
Society.java:82: error: illegal start of expression
  public Person getOldestPerson () {
  ^
Society.java:82: error: ';' expected
  public Person getOldestPerson () {
                               ^
Society.java:93: error: reached end of file while parsing
}

**Comment by bradwray** [comment]
Nice structure but it looks like people's personality numbers are always 0. Keep on trying.

**Comment by thebestburrito** [comment]
So, I kept working on getting the booleans and names to work with the
personality numbers, and I kept running into the problem that one of my
methods, the number generator, is referencing a non-static field in a
static manner. So I figured that a good way to fix this is to change all
the booleans and the string to static fields, and the corresponding methods
to static as well. This got rid of the error messages, but ran into
problems when I ran main. The numbers were correctly and randomly
assigning, but the corresponding names were incorrectly assigning. I did
not know how to fix this, since the only way that the booleans and the
string personalityString are initialized is through the setter and the
generator. Is there a way that you can look at my most recent commit? I am
sorry that I am not explaining this too well. I want to get my stuff out so
others can work with it, but I don't want to send out some janky, broken,
POS that is neither functional or helpful. I am also sorry to bother you
over the weekend for this.

**Pull request closed without merging**